src/pages/policy.jsx

```diff
@@ -1,10 +1,99 @@
 import React from "react";
 import PageLayout from "../common/layout/PageLayout";
+import { H1, H4, P } from "../common/components/elements/Text";
 
 export default function Policy() {
   return (
-    <PageLayout>
-      <div>Policy page...</div>
+    <PageLayout className="text-primary-900 items-stretch">
+      <H1 className="mb-5">Terms of Use</H1>
+      <div className="text-justify mb-5">
+        <H4 className="font-medium">Welcome to PlaceXP!</H4>
+        <P className="">{"This website is a multi-utility platform to facilitate all the VITians timely access to all the academic related resources and solve any queries related to the curriculum or co-curriculum, for exploring opportunities and enhancing outreach using the online search and other tools available for all the VITians. It is owned and managed by the team of PlaceXP as a registered club of Vellore Technology, Chennai, 600127(India)."}</P>
+        <P className="">{"In order for all of our users to enjoy the PlaceXP Services, it is important to follow the terms and conditions described in this Terms of Use (`Terms of Use` or `Terms`). By visiting the PlaceXP Site (whether or not you are a registered user) or using any of the PlaceXP Services on any of the Sites, you are accepting this Terms of Use. Please take a moment to carefully read through these Terms. It is our goal to provide you with an excellent user experience, so if you have any questions or comments about these Terms, please contact us at placexpvitc@gmail.com."}</P>
+      </div>
+
+      <div className="text-justify mb-5">
+        <H4 className="font-medium">Accepting The Terms:</H4>
+        <ol className="list-decimal my-2 ml-5">
+          <li><P className="">{"In order to use the Service, you must firstly agree to the Terms. You may not use the Service if you do not accept the Terms."}</P></li>
+          < li > <P className="">{"You can accept the Terms by simply using the Service. You understand and agree that PlaceXP will treat your use of the Service as acceptance of the Terms from that point onwards."}</P></li>
+          < li > <P className="">{"You may not use the Service and may not accept the Terms if (a) you are not of legal age to form a binding contract with PlaceXP, or (b) you are a person who is either barred or otherwise legally prohibited from receiving or using the Service under the laws of the country in which you are resident or from which you access or use the Service."}</P></li>
```

Comment:
no gap should be there
```suggestion
          <li><P>{"You can accept the Terms by simply using the Service. You understand and agree that PlaceXP will treat your use of the Service as acceptance of the Terms from that point onwards."}</P></li>
          <li><P>{"You may not use the Service and may not accept the Terms if (a) you are not of legal age to form a binding contract with PlaceXP, or (b) you are a person who is either barred or otherwise legally prohibited from receiving or using the Service under the laws of the country in which you are resident or from which you access or use the Service."}</P></li>
```